### DISCUSSION TITLE
Someone knows If its possible implement this in a raspberry pi?

### DISCUSSION THREAD

**Discussion by luis71** [Live-Charts/LiveCharts2#291] [Q&A]
Context: I have to make a UI, that comminicate Arduino Mega with a Raspberry pi (serialport), the UI have to configure some parameters, calculate some equations (trigonometry), graph values and if its possible with the data put all together in a excel file.

Also:
-I never used Visual Studio before.
-I have one UI but I did it with Matlab app designer, so as the Raspberry works with ARM doesn't works. Now I'm trying with Visual Studio (winforms)
-I want to run it on a Raspberry Pi (model 4B with Raspberry Pi OS but I don't mind change the OS).
-I'm new working with Raspberry pi (I try with eric but was difficult to me make there the UI and the final view don't like me).
-I read that its possible open visual studio winForms in the Raspberry with mono but also I used two NuGet packages (MaterialSkin.2 and LiveCharts packages in VS)
![image](https://user-images.githubusercontent.com/25520204/142350764-79a3d0c3-4f23-46a6-9999-130e877dee07.png)
when I try to open the UI appears some errors
![image](https://user-images.githubusercontent.com/25520204/142350783-66e3f5cd-8833-4308-857c-bc7962fbdf8a.png)
3 versions of the program (using only one package and then both) so I don't know what can I do or even if it's possible to open that files.

I have another simple UI using only the visual studio tools and works fine (with mono) 
![image](https://user-images.githubusercontent.com/25520204/142350799-8b3a050c-7244-4b04-a7a9-e7fa855fcf58.png)
 then I made 3 compile versions of the UI (the one I want to use) using only one package and then both, marked with red each one (image 2), the question is... can I run the UI using that two packages? if I can't can someone recommend me some programm to make a native UI for Raspberry pi wit an nice final look, like google form or something like that
![image](https://user-images.githubusercontent.com/25520204/142350824-a3868530-eb11-4a29-b547-65c29d8a5b80.png)
UI main screen that I want to run (without data in the graphs).

**Comment by preintel** [comment]
Im not sure if it is possible to run it on rasberry pi but seems like you are getting an error for missing reference:
System.Windows.Forms.Integration

try adding this to the top of each C# class that uses cartesianchart:
using System.Windows.Forms;
using System.Windows.Forms.Integration;

If that does not work and you get the same error regarding WindowsFormIntegration, let me know what version of .net is running on the raspberry pi. WindowsFormIntegration is needed for integrating controls between different formats such as Winforms/WPF/etc and LiveCharts2 works across these.
